lib/plugins/create/templates/aws-fsharp/build.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
+isMacOs=`uname -a | grep Darwin`
+
 #install zip
-apt-get -qq update
-apt-get -qq -y install zip
+if [ -z  isMacOs ]
```

Comment:
Shouldn't this be a negation? So "if it's not MacOS, then update and install `zip`"?

I get an error that `zip` is not installed when running this in a Docker container.